bin/make_cfgs.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+import re
+import json
+import argparse
+import os
+import network_manager
+
+description = """ 
+Generates configs based on templates for services and pelagos configuration file
+
+for dnsmasq generates a file 
+    <target dir>/etc/dnsmasq/dnsmasq.d/network_nodes.conf
+with records like this
+ 
+    ...
+    dhcp-host=aa:bb:cc:dd:68:7c,ses-client-3,1.2.3.13
+    ...
+    ptr-record=13.3.2.1.in-addr.arpa,ses-client-3.a.b.de
+    ...
+
+for conman generates file <target dir>/states/etc/conman.conf  based on 
+template  <target dir>states/etc/conman.conf.tmpl with records like this
+
+    CONSOLE name="ses-client-3" IPMIOPTS="U:<user>,P:<pass>" dev="ipmi:<ipmi server>"
+
+for salt generates roster file <target dir>/deploy.roster
+
+    ses-client-3.a.b.de:
+      host: 1.2.3.13
+      user: root
+
+for teuthology generates sql script <target dir>/sql/nodes.sql
+
+    INSERT INTO nodes (name, machine_type, mac_address, is_vm, locked, arch, description)
+        VALUES  ( 'ses-client-3.a.b.de' , 'client' ,
+            'aa:bb:cc:dd:67:16' , 'f' , 'f' , 'x86-64' , '' ) ;
+"""
+
+parser = argparse.ArgumentParser(description=description,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+
+
+def reorder_ip(ip):
```

Comment:
use reverse?

Reply:
and it was renamed, you could see in branch